### PR TITLE
MEPTS-534: DSD Indicator - N9

### PR DESCRIPTION
**Pull request by rubailly**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java [hunk 1]
```diff
@@ -715,7 +712,7 @@ private CohortDefinition getPatientsWhoCompletedRapidFlow() {
     CodedObsCohortDefinition cd = new CodedObsCohortDefinition();
     cd.addParameter(new Parameter("onOrBefore", "Before Date", Date.class));
     cd.addParameter(new Parameter("onOrAfter", "After Date", Date.class));
-    cd.addParameter(new Parameter("locationList", "Location", Location.class));
+    cd.addParameter(new Parameter("location", "Location", Location.class));
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java:715]: @simaomazive why did you change this?

Comment (simaomazive) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java:715]: I made these changes in order to be able to run livetst for N9. But I will put it back. I forgot
